stack_examples/aws_ssm_package_installer/main.tf

```diff
@@ -0,0 +1,141 @@
+resource "turbot_policy_setting" "ssmStackSource" {
+  resource = var.region_resource
+  type     = "tmod:@turbot/aws-ssm#/policy/types/ssmStackSource"
+  value    = <<POLICY_VALUE
```

Comment:
Nathan prefers that we do not use HEREDOC like we have here. He prefers 

EOx where x is anything

Reply:
Typically most examples have EOF